x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -292,11 +289,8 @@ export class FleetPlugin
           },
           authz: await getAuthzFromRequest(request),
           epm: {
-            // Use a lazy getter to avoid constructing this client when not used by a request handler
-            get internalSoClient() {
-              return appContextService
-                .getSavedObjects()
-                .getScopedClient(request, { excludedWrappers: ['security'] });
+            get internalSoClient(): ISavedObjectsRepository {
```

Comment:
Should we rename to this to `savedObjectRepository`? I think `internal` is a little redundant since repository is always internal.